### PR TITLE
Create standards-assurance-tables.md

### PR DESCRIPTION
**Pull request by joelgsamuel**
Publish the SAT template/method with surrounding guidance which can be referenced

### CODE REVIEW

File: guides/standards-assurance-tables.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+expires: 2019-09-01
+---
+# Standards Assurance Tables
+
+The MOJ Cyber Security team have developed a 'Standards Assurance Table' (SAT) in the form of a Google Sheet template.
+
+The SAT measures technology systems (and surrounding information governance) against the [UK Cabinet Office](https://www.gov.uk/government/organisations/cabinet-office) [Minimum Cyber Security Standard (MCSS)](https://www.gov.uk/government/publications/the-minimum-cyber-security-standard) and [UK National Cyber Security Centre (NCSC)](https://www.ncsc.gov.uk/) [Cloud Security Principles (CSPs)](https://www.ncsc.gov.uk/collection/cloud-security?curPage=/collection/cloud-security/implementing-the-cloud-security-principles).
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: measures -> enables comparison between

File: guides/standards-assurance-tables.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+For transparency and open-working purposes, a [redacted copy of the Standards Assurance Table](https://docs.google.com/spreadsheets/d/1oO9RAZQ6ETsN2iHZb5zcFM0HROliw18MBv89oF-BPeg/edit?usp=sharing) has been published. Please note, this is not the functional template used within the MOJ.
+
+## SAT Template
+
+The SAT itself is written to be self-explanitory to a cyber security professional that is already aware of the MCSS/CSP and has a familiarity with information risk management concepts.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: explanitory -> explanatory
that is -> who is

File: guides/standards-assurance-tables.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+- Black labelled sheets describe the SAT and how it should be used
+- Blue labelled sheets are the sheets to be completed
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: sheets to be completed -> ones to complete

File: guides/standards-assurance-tables.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+- Yellow labelled sheets are automatically calculated reporting based on the blue labelled sheet data
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: calculated reporting -> calculated, providing reports

File: guides/standards-assurance-tables.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+- Green labelled sheets offer help/guidance on SAT components
+
+## Key SAT concepts
+
+The SATs have concepts of "Objectives", "Evidence", "Confidence", an overall "Delta" (which is the most pertinent output) and "Further Evidence Required", with supporting commentary.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: concepts of -> measures including

File: guides/standards-assurance-tables.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+The primary SAT purpose is to assess the system against the MCSS/CSP, determining how compliant (or non-compliant) the system is, based on a collation and review of evidence. Evidence is analysed to determine confidence that the evidence demonstrates the system meets (or does not meet) the standard, and the 'gap' (delta) between system posture according to said evidence and the standards.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: to assess -> to help assess
MCSS/CSP, determining -> MSS/CSP. It is used to determine
to determine confidence that the evidence demonstrates the system -> to demonstrate and provide evidence that the system
standard, and the 'gap' -> standard. It also indicates the 'gap'
between system -> between the system

File: guides/standards-assurance-tables.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+
+### Objectives
+
+The MCSS/CSPs have been distilled into 39 Objectives which the assessor (typically a cyber security professional) completing the SAT is measuring against.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: Objectives which the assessor (typically a cyber security professional) completing the SAT is measuring against. -> The Assessor (normally a cyber security professional) completes the SAT by evaluating the target system against the objectives.

File: guides/standards-assurance-tables.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+
+The [categories used within the MCSS](https://ministryofjustice.github.io/security-guidance/principles/identify-protect-detect-respond-recover/#identify-protect-detect-respond-recover) have been discussed seperately.
+
+Objectives are templated, so can be added to but not modified in-place.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: templated, so can -> templated. This means they can

File: guides/standards-assurance-tables.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+
+### Evidence
+
+To avoid assessments that are ultimately anecdotal, the SAT considers written evidence.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: considers -> builds upon
written -> documentary

File: guides/standards-assurance-tables.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+
+Evidence can come in the form of transcribed conversations, diagrams, documentation or an otherwise written conveyable information about a system.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: an otherwise written conveyable -> other auditable

File: guides/standards-assurance-tables.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+Objectives are templated, so can be added to but not modified in-place.
+
+### Evidence
+
+To avoid assessments that are ultimately anecdotal, the SAT considers written evidence.
+
+Evidence can come in the form of transcribed conversations, diagrams, documentation or an otherwise written conveyable information about a system. 
+
+Evidence may not be related to the system itself but form a part, for example, where there is a wider document that is not system orientated that describes who relevant role holders currently are.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: Evidence may not -> Evidence might not
be related -> be directly related
orientated that describes who relevant -> orientated but which describes who is relevant

File: guides/standards-assurance-tables.md [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+
+Evidence is 'Held', 'Partial', 'Not Held' or 'N/A' (where the Objective is not applicable to the system being assessed).
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: Evidence is 'Held', -> Evidence is described as being 'Held',

File: guides/standards-assurance-tables.md [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+
+### Confidence
+
+The assessor reviews the evidence and uses their professional opinion to indicate a Confidence Score.
+
+The Confidence Score is based on a scale of 0 (no confidence at all) to 14 (high level of confidence), or 'N/A' (where the Objective is not applicable to the system being assessed).
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: is based on a scale of -> uses a scale from

File: guides/standards-assurance-tables.md [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+
+### Delta
+
+The Delta Rating is the resulting 'distance' between the assessed system posture against an Objective and the confidence of the same.
+
+Mathematically, the results are N/A (where the Objective is not applicable to the system being assessed) or 0 to 14 (inc).
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: results are -> final Delta Rating is

File: guides/standards-assurance-tables.md [hunk 15]
```diff
@@ -0,0 +1,158 @@
...
+
+A wide delta (higher numerical value) conveys the Objective is not met. A narrow delta (lower numerical value) conveys the Objective is closer to being met.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: conveys -> indicates that
(both instances!)

File: guides/standards-assurance-tables.md [hunk 16]
```diff
@@ -0,0 +1,158 @@
...
+
+The Delta Rating is automatically calculated as '14 minus Confidence Score'.
+
+### Further Evidence Required
+
+The assessor indicates what further evidence _types_ in their view are required based on the evidence they have thus far.
+
+The [Further Evidence Required (Help) sheet](https://docs.google.com/spreadsheets/d/1oO9RAZQ6ETsN2iHZb5zcFM0HROliw18MBv89oF-BPeg/edit#gid=679100590) has a calculator which the assessor will use.
+
+This data point was designed as unique numericals as this allows programmatic analysis in the future. This data point and how it is formed is currently under active review.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: This data point was designed as unique numericals as this allows programmatic analysis in the future. This data point and how it is formed is currently under active review. -> The data point is currently a unique number to assist with future automated analysis. The format and range of values for the data point is currently under active review and so subject to change without notice.

File: guides/standards-assurance-tables.md [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+
+## Understanding the Objectives, gathering evidence for the assessor
+
+Teams/individuals responsible for the design, creation, implementation, support and maintenance of systems should have viable written evidence (regardless of format) that should be made available to various teams on request, for example, security or to internal audit.
+
+Using the [categories used within the MCSS](https://ministryofjustice.github.io/security-guidance/principles/identify-protect-detect-respond-recover/#identify-protect-detect-respond-recover) as sections some helpful thought questions and documentation expectations are discussed below.
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: as sections some helpful thought -> as a basis, some indicative

File: guides/standards-assurance-tables.md [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+- Team organisation charts
+- Data Privacy Impact Assessments (DPIAs)
+- Information risk management documentation (for example, RMADS)
+- Information flow diagrams
+
+#### Thought questions
+
+- Who is responsible and/or accountable for the the system whether from an operational or budgetary perspective?
+- Who is responsible and/or accountable for the information held inside the system?
+- What security-focused work has been conducted on any suppliers and supplier systems to ensure they are safe for use/integration?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: conducted -> conducted recently (within the last year)

File: guides/standards-assurance-tables.md [hunk 19]
```diff
@@ -0,0 +1,158 @@
...
+- Where is the system technically hosted?
+- Where are the places where the system stores data?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: Where are the places where the system stores data? -> In what services or geographical locations does the system store data?

File: guides/standards-assurance-tables.md [hunk 20]
```diff
@@ -0,0 +1,158 @@
...
+- What are the consequences if the system is unavailable to users or data has been lost/corrupted?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: Add:
How do the consequences of unavailability change over time? For example, after one hour, one day, one week, one month... permanent.

File: guides/standards-assurance-tables.md [hunk 21]
```diff
@@ -0,0 +1,158 @@
...
+- What happens for business continuity / disaster recovery if the system is unavailable or data has been lost/corrupted?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: What happens for business continuity / disaster recovery -> What changes - if anything - regarding business continuity / disaster recovery processes or plans

File: guides/standards-assurance-tables.md [hunk 22]
```diff
@@ -0,0 +1,158 @@
...
+- Penetration test / IT Health Check reports and remedial documentation
+- Risk registers
+
+#### Thought questions
+
+- How does the system ensure only authorised people can use the system?
+- How are system users managed for joiners, movers and leavers?
+- How is the system's underlying software kept up to date for security software patching?
+- How does the system protect itself appropriately and proportionately from attackers?
+- How do ensure the system can protect itself from attackers over time, so it is secure now but also will remain secure in the future?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: How do ensure the -> What assurance is there that the

File: guides/standards-assurance-tables.md [hunk 23]
```diff
@@ -0,0 +1,158 @@
...
+- How often has technical security testing been conducted? Where within the system?
+- How does the system stay up to date using modern encryption to keep data safe?
+- Does the system use multi-factor authentication (MFA, also known as 2FA)?
+- For people who have access to the system, do they have all the right clearances in place?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: in place? -> in place? How is this assured?

File: guides/standards-assurance-tables.md [hunk 24]
```diff
@@ -0,0 +1,158 @@
...
+
+- Information risk management documentation (for example, RMADS)
+- Technical/system architecture documentation
+- Penetration test / IT Health Check reports and remedial documentation
+- Risk registers
+
+#### Thought questions
+
+- How does the system, and accompanying operational support teams, know/detect when the system is under attack?
+- How is access to the system (both legitimate and illegitimate) logged so retrospective investigations can take place to determine 'who did what when'?
```
Comment (warmanaMOJ) [inline guides/standards-assurance-tables.md]: Add:
How is the required level of detail in logs determined? How long are log files retained?

File: guides/standards-assurance-tables.md (hunk 1 above)
Comment (joelgsamuel) [reply guides/standards-assurance-tables.md]: I don't think it is comparison against, it is measurement for our purposes?

**Pull request merged**